ui/puzzle/src/interfaces.ts

```diff
@@ -52,9 +54,11 @@ export interface Controller extends KeyboardController {
   getNode(): Tree.Node;
   showComputer(): boolean;
   trans: Trans;
-  getData(): PuzzleData;
+  data: PuzzleData;
   getTree(): TreeWrapper;
   ground: Prop<CgApi | undefined>;
+  chessground: CgApi;
```

Comment:
Adding `chessground` when there is already a `ground()` prop feels redundant/hacky